workflows/gmsemu.nf

```diff
@@ -190,7 +190,7 @@ workflow GMSEMU {
     ch_versions = ch_versions.mix(GENERATE_MASTER_HTML.out.versions.first())
 
     emit:
-    nanostats_unprocessed   = NANOPLOT_UNPROCESSED_READS.out.txt // channel: [ path(versions.yml) ]
```

Comment:
I think it would be nicer to just move the line where it's called (L133) to Line 113. I should've put it there in the first place. Then no need to do hacky emits:)